portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -15,18 +15,32 @@
 package com.google.sps.servlets;
 
 import java.io.IOException;
+import com.google.gson.Gson;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.*;
```

Comment:
Please don't use wildcars imports. I don't know why reformater didn't fix it.
https://google.github.io/styleguide/javaguide.html#s3.3.1-wildcard-imports

Reply:
I did not know about that! Thank you very much for the link